chi/lease.py

```diff
@@ -697,6 +815,15 @@ def lease_duration(days=1, hours=0):
 # Leases
 #########
 
+def list_leases() -> List[dict]:
```

Comment:
This should return `List(Lease)`, with the JSON import, this should be pretty straightforward. I see doing `get_lease` this way would break backwards compatibility. Let's still make the change, and I'll see about an integration plan for trovi that is OK with this.